packages/aws-lambda-auto-wrap/esm/index.js

```diff
@@ -6,16 +6,13 @@
 
 const instana = require('@instana/aws-lambda');
 const localUtils = require('./utils');
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
```

Comment:
FYI A good practice is to group requires/imports. First the external libs and then the local libraries. It improves readability.